src/finality/justification/decode.rs

```diff
@@ -54,6 +54,19 @@ pub struct Justification {
     // TODO: pub votes_ancestries: Vec<Header>,
 }
 
+impl<'a> From<&'a Justification> for JustificationRef<'a> {
+    fn from(j: &'a Justification) -> JustificationRef<'a> {
+        JustificationRef {
+            round: j.round,
+            target_hash: &j.target_hash,
+            target_number: j.target_number,
+            precommits: PrecommitsRef::Decoded(&j.precommits),
+            // todo
```

Comment:
```suggestion
            // TODO:
```

This is pretty nit-picky, but all the code uses uppercase `TODO:` at the moment